modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl

```diff
@@ -237,7 +237,7 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 		<#if propertySchema.jsonMap>
 			@JsonAnySetter
 		</#if>
-		<#if propertySchema.jsonString>
+		<#if freeMarkerTool.isVersionCompatible(configYAML, 2) && propertySchema.jsonString>
```

Comment:
Should this be `4`? [We are already on 3](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/RESTBuilder.java#L355-L362). @ccorreagg what do you think?

Reply:
In that case we need to add the compatibilityVersion to the workflow module.

Reply:
I will update to 3, thanks guys!

Reply:
Nop, because I guess they won't have the `x-json-string` set to true

Reply:
The condition is an and. They must have set the property `x-json-string: true` and `compatibilityVersion: x`

Reply:
But up to you, guys. If you want to set the property to `4` as it is a new value, the actions classes will be created for all the Liferay portal. Other option would be setting 4 for this functionality and upgrading 4 to 5 for the actions

Reply:
Should we maintain the compatibilityVersion to 2 and if this functionality needs more requirement we can upgrade it, taking into account the `actions` ones?

Reply:
As decided offline: we will upgrade actions to 4 and use 3 for this.